src/WebCamera.xml

```diff
@@ -78,45 +15,106 @@
         </property>
         <property key="captureButtonIcon" type="string" defaultValue="camera">
             <caption>Capture icon</caption>
-            <category>Button captions</category>
+            <category>General</category>
             <description>The name of the glyph-icon to use for capture feature.</description>
         </property>
+        <property key="usePictureButtonName" type="translatableString" required="false">
+            <caption>Save button</caption>
+            <category>General</category>
+            <description>This button acts as the OK/save/submit button. This button is hidden in capture mode.</description>
+            <translations>
+                <translation lang="en_US">Save</translation>
+                <translation lang="nl_NL">Opslaan</translation>
+            </translations>
+        </property>
         <property key="switchCameraIcon" type="string" defaultValue="refresh">
             <caption>Switch camera icon</caption>
-            <category>Button captions</category>
+            <category>General</category>
             <description>The name of the glyph-icon to use for switch camera. </description>
         </property>
         <property key="usePictureButtonIcon" type="string" defaultValue="download">
```

Comment:
Change to `translatableString`

Reply:
the values refresh, download and camera are used for glyph icons